go.mod

```
@@ -122,6 +122,7 @@ require (
 	go.opentelemetry.io/otel/sdk v1.11.2
 	go.opentelemetry.io/otel/trace v1.11.2
 	go.opentelemetry.io/proto/otlp v0.19.0
+	golang.org/x/crypto v0.2.0
```

```suggestion
	golang.org/x/crypto v0.2.0 // replaced
```

so it's easier to spot when we do dependency updates. Same for api.